docs/v0.4/getting-started/docker-for-windows.md

```diff
@@ -181,38 +188,105 @@ NAME     LATEST IMAGE
 square   index.docker.io/$DOCKER_ID/square@sha256:ac089ca183368aa831597f94a2dbb462a157ccf7bbe0f3868294e15a24308f68   square.js   <empty>   <empty>   Ready    1m13s
 ```
 
-## create a deployer
+> Due to differences between Windows and Linux file permissions, local builds are not supported on Windows. We suggest building from a git repository as a workaround.
```

Comment:
this line is out of context since there is nothing talking about local builds

Reply:
It's in the part where builds happen and we could have used a local build. Seems more context that at the end no?

Reply:
agree that it seems a bit out of place, could we change the heading to be "create a function"  and add this note after the heading, that way it is more in context